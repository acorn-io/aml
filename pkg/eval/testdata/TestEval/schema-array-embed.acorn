schema {
    a: {
        [number, string]
    }
}

a: ["hi"]
