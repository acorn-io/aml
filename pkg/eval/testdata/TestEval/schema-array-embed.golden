`schema violation .a: expected kind object but got array with value (["hi"]) (schema-array-embed.acorn:2:8) (schema-array-embed.acorn:1:8)`
